model/publishing/PublishingService.php

```diff
@@ -80,10 +82,11 @@ public function addSlashes($values)
         if (isset($values[PublishingService::PUBLISH_ACTIONS])) {
             if (is_array($values[PublishingService::PUBLISH_ACTIONS])) {
                 $values[PublishingService::PUBLISH_ACTIONS] = array_map(function($item) {
-                    return addslashes($item);
+                    $a = addslashes($item);
+                    return str_replace('\\\\\\\\', '\\\\', addslashes($item));
```

Comment:
Could u find a nicer way to do that?

Reply:
there was a bug, I've fixed it, you can provide your decision :)